rules/Rglpk.json

```diff
@@ -0,0 +1,35 @@
+{
+  "patterns": [
+    "\\bRglpk\\b"
```

Comment:
It's better for this pattern to be generic, so it matches any R package (not just Rglpk) that needs the GLPK library. How about `\\bglpk\\b`?

Similarly, I'd suggest naming this rule file `glpk.json`.